app/mqtt.mjs

```diff
@@ -140,14 +161,14 @@ export const configureMqttDiscovery = async (modbusClient, mqttClient) => {
         'exhaustAirHumidity': createHumiditySensorConfiguration(configurationBase, 'exhaustAirHumidity', 'Exhaust air humidity'),
         'mean48HourExhaustHumidity': createHumiditySensorConfiguration(configurationBase, 'mean48HourExhaustHumidity', 'Exhaust air humidity (48h mean)'),
         // Generic sensors (percentages, minutes left, cascade values)
-        'heatRecoverySupplySide': createSensorConfiguration(configurationBase, 'heatRecoverySupplySide', 'Heat recovery (supply)', {'unit_of_measurement': '%'}),
-        'heatRecoveryExhaustSide': createSensorConfiguration(configurationBase, 'heatRecoveryExhaustSide', 'Heat recovery (exhaust)', {'unit_of_measurement': '%'}),
+        'heatRecoverySupplySide': createSensorConfiguration(configurationBase, 'heatRecoverySupplySide', 'Heat recovery (supply)', { 'unit_of_measurement': '%' }),
+        'heatRecoveryExhaustSide': createSensorConfiguration(configurationBase, 'heatRecoveryExhaustSide', 'Heat recovery (exhaust)', { 'unit_of_measurement': '%' }),
         'cascadeSp': createSensorConfiguration(configurationBase, 'cascadeSp', 'Cascade setpoint'),
         'cascadeP': createSensorConfiguration(configurationBase, 'cascadeP', 'Cascade P-value'),
         'cascadeI': createSensorConfiguration(configurationBase, 'cascadeI', 'Cascade I-value'),
-        'overPressureTimeLeft': createSensorConfiguration(configurationBase, 'overPressureTimeLeft', 'Overpressure time left', {'unit_of_measurement': 'minutes'}),
-        'ventilationLevelTarget': createSensorConfiguration(configurationBase, 'ventilationLevelTarget', 'Ventilation level (target)', {'unit_of_measurement': '%'}),
-        'ventilationLevelActual': createSensorConfiguration(configurationBase, 'ventilationLevelActual', 'Ventilation level (actual)', {'unit_of_measurement': '%'}),
+        'overPressureTimeLeft': createSensorConfiguration(configurationBase, 'overPressureTimeLeft', 'Overpressure time left', { 'unit_of_measurement': 'minutes' }),
+        'ventilationLevelTarget': createSensorConfiguration(configurationBase, 'ventilationLevelTarget', 'Ventilation level (target)', { 'unit_of_measurement': '%' }),
+        'ventilationLevelActual': createSensorConfiguration(configurationBase, 'ventilationLevelActual', 'Ventilation level (actual)', { 'unit_of_measurement': '%' }),
```

Comment:
There are some unrelated whitespace changes here, can you undo them? I should add a formatter to the project.

Reply:
Actually, let them be. We'll format the whole thing later with `prettier`.

Reply:
The whole thing is formatted with prettier, as all the other documents touched here.